src/joint.dia.link.js

```diff
@@ -1230,89 +1226,89 @@ joint.dia.LinkView = joint.dia.CellView.extend({
 
     pointerdown: function(evt, x, y) {
 
+
         joint.dia.CellView.prototype.pointerdown.apply(this, arguments);
+        this.notify('link:pointerdown', evt, x, y);
 
         this._dx = x;
         this._dy = y;
 
-        var interactive = _.isFunction(this.options.interactive) ? this.options.interactive(this, 'pointerdown') : this.options.interactive;
-        if (interactive === false) return;
-
-        var className = evt.target.getAttribute('class');
-        var parentClassName = evt.target.parentNode.getAttribute('class');
-        var labelNode;
-        if (parentClassName === 'label') {
-            className = parentClassName;
-            labelNode = evt.target.parentNode;
-        } else {
-            labelNode = evt.target;
-        }
+        // if are simulating pointerdown on a link during a magnet click, skip link interactions
+        if (evt.target.getAttribute('magnet') == null) {
```

Comment:
Can we change this to `if (evt.target.getAttribute('magnet') != null) return;` to prevent unnecessary indenting?